playbook/app/pb_kits/playbook/pb_table/docs/example.yml

```diff
@@ -33,7 +34,7 @@ examples:
     - table_md: Medium
     - table_lg: Large
     - table_sticky: Sticky Header
-    - table_alignment_row: Row Alignment
```

Comment:
I think this has accidentally replaced the Row Alignment doc ex in the review env, instead of putting the Sticky Left Column doc ex ahead of it in the list.